### ISSUE TITLE
json reader for glTF

### ISSUE THREAD

**Issue by acrlakshman** [9prady9/MeshIO#48]
Since glTF adopts json format, can we try to use https://github.com/nlohmann/json to parse `.gltf` files.

How should we parse .bin files?

Ref.: @9prady9

**Comment by acrlakshman** [comment]
I just realized [tinygltf](https://github.com/syoyo/tinygltf) uses the json library I mentioned above.

**Comment by 9prady9** [comment]
Yeah, I have used https://github.com/nlohmann/json once before, it is very nice I think. It makes reading json a super easy task. We can use it for reading json. I created a two-people chat room for which you should have got invite link. Join that, it will make discussions easy.
